configs/common/modules_lmod.yaml

```diff
@@ -234,7 +234,7 @@ modules:
         environment:
           set:
             'WGRIB2_INC': '{prefix}/include'
-            'WGRIB2_LIB': '{prefix}/lib/libwgrib2.a'
+            'WGRIB2_LIB': '{prefix}/lib64/libwgrib2.so'
```

Comment:
This is not a comment for this PR per se, but we could consider defining this dynamically in the Spack recipe so that it reflects actual compiled library path. Either way this is fine right now.